back-end/helpers/transcript_upload_interactor.js

```diff
@@ -0,0 +1,28 @@
+import {InputBoundaryInterface} from "../interfaces/input-boundary-interface.js";
```

Comment:
This file should probably just be called transcript_interactor (since it does not only deal with uploads)